tests/parameters/hydrogels_one_field_01.prm

```diff
@@ -0,0 +1,151 @@
+end
+subsection Domain
+  set Colorize                      = true
+  set Copy boundary to manifold ids = false
+  set Copy material to manifold ids = false
+  set Create default manifolds      = true
+  set Grid to generate              = rectangle
+  set Input grid file name          =
+  set Manifold descriptors          =
+  set Mesh smoothing alogrithm      = none
+  set Optional Point<spacedim> 1    = 0,0,0
+  set Optional Point<spacedim> 2    = 1,1,1
+  set Optional double 1             = 1.0
+  set Optional double 2             = 0.5
+  set Optional double 3             = 1.5
+  set Optional int 1                = 1
+  set Optional int 2                = 2
+  set Optional vector of dim int    = 1,1,1
+  set Output grid file name         =
+end
+subsection Error Tables
+  set Compute error            = true
+  set Error file format        = tex
+  set Error precision          = 3
+  set Output error tables      = true
+  set Solution names           = u,u,u
+  set Solution names for latex = u,u,u
+  set Table names              = error
+  set Write error files        = false
+  subsection Table 0
+    set Add convergence rates          = true
+    set Extra terms                    = cells,dofs
+    set Latex table caption            = error
+    set List of error norms to compute = L2,H1;L2,H1;L2,H1
+    set Rate key                       =
+  end
+end
+subsection Exact solution
+  set Function constants  =
+  set Function expression = 0; 0; 0
+  set Variable names      = x,y,z,t
+end
+subsection Forcing terms
+  set IDs and component masks =
+  set IDs and expressions     =
+  set Known component names   = u,u,u
+  set Used constants          =
+end
+subsection Free Swelling One Field
+  set Block of differential components = 1
+  set Blocking of the finite element   = u,u,u
+  set Finite element space             = FESystem[FE_Q(2)^d]
+  set G                                = 1e4
+  set Omega                            = 1e-5
+  set T                                = 298.0
+  set chi                              = 0.1
+  set l0                               = 1.5
+end
+subsection IDA Solver Parameters
+  set Absolute error tolerance                      = 1e-4
+  set Final time                                    = 1
+  set Ignore algebraic terms for error computations = false
+  set Initial condition Newton max iterations       = 5
+  set Initial condition Newton parameter            = 0.33
+  set Initial condition type                        = use_y_diff
+  set Initial condition type after restart          = use_y_dot
+  set Initial step size                             = 1e-4
+  set Initial time                                  = 0
+  set Maximum number of nonlinear iterations        = 20
+  set Maximum order of BDF                          = 5
+  set Min step size                                 = 5e-5
+  set Relative error tolerance                      = 1e-3
+  set Seconds between each output                   = 1e-1
```

Comment:
ida stima la dinamica del sistema in base questo parametro. se vuoi che ida usi passi più piccoli riduci questo valore